sources/models/shared-defaults/oci-capabilities.toml

```diff
@@ -0,0 +1,17 @@
+[settings.oci-defaults.capabilities]
+# These values were sourced from the output of `ctr oci spec`, then
+# modified for Bottlerocket.
```

Comment:
"These values represent the default capabilities in the default OCI spec for containerd, except for CAP_NET_RAW , which is disabled by default in Bottlerocket".

^ comment in file

comment in PR explains how we got the default OCI spec values